scipy/sparse/linalg/isolve/tests/test_lgmres.py

```diff
@@ -91,7 +91,7 @@ def test_outer_v(self):
     @pytest.mark.skipif(python_implementation() == 'PyPy',
                         reason="Fails on PyPy CI runs. See #9507")
     def test_arnoldi(self):
-        np.random.rand(1234)
```

Comment:
I'm 99% sure this should have been `random.seed(1234)` see other methods that start with this.